### DISCUSSION TITLE
Debian12 - Akvorado dont show Flows on UI

### DISCUSSION THREAD

**Discussion by noll26** [akvorado/akvorado#1182] [Q&A]
I have install Akvorado using quick start guide. I removed the demo datas and all is looking good but no have flow data in UI. 

Using tcpdump on Debian12 i can see the received flows from my Juniper Router but in Akvorado web view its dont see nothing.

noll@debian:~/akvorado# tcpdump port 2055 -n
tcpdump: verbose output suppressed, use -v[v]... for full protocol decode
listening on enp6s18, link-type EN10MB (Ethernet), snapshot length 262144 bytes
06:40:20.382885 IP X.X.X.X.57240 > 192.168.200.3.2055: UDP, length 1416
06:40:20.383077 IP X.X.X.X.57240 > 192.168.200.3.2055: UDP, length 1416
06:40:20.383359 IP X.X.X.X.57240 > 192.168.200.3.2055: UDP, length 1416
06:40:20.383635 IP X.X.X.X.57240 > 192.168.200.3.2055: UDP, length 1416
06:40:20.383809 IP X.X.X.X.57240 > 192.168.200.3.2055: UDP, length 1416

![image](https://github.com/akvorado/akvorado/assets/31144703/fd7834d8-2161-4e74-85d8-b542edabdd3e)

**Comment by alifgufron** [comment]
See Troubleshooting section:

http://avocado.melsa.net.id:8081/docs/troubleshooting#no-packets-exported

and ensure  flushing the conntrack table

```
conntrack -D -p udp --orig-port-dst 2055
```
